tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+	"compilerOptions": {
+		"noEmit": true,
+		"strict": true,
+		"jsx": "react",
+		"target": "esnext",
+		"module": "esnext",
+		"moduleResolution": "node",
+		"esModuleInterop": true,
+		"removeComments": true,
+		"baseUrl": "./",
+		"paths": {
+			"@clay/*": ["packages/*/src"],
```

```suggestion
			"@clayui/*": ["packages/*/src"],
```

To match the change you made elsewhere? AFAIK we've requested `@clay` but no idea whether that can/will happen.